crates/biome_js_analyze/src/lint/style/use_import_type.rs

```diff
@@ -318,20 +318,24 @@ impl Rule for UseImportType {
             ImportTypeFix::RemoveTypeQualifiers(type_tokens) => {
                 let mut diagnostic = RuleDiagnostic::new(
                     rule_category!(),
-                    import.import_clause().ok()?.type_token()?.text_trimmed_range(),
-                    "The import has this type qualifier that makes all inline type qualifiers useless.",
+                    import_clause.type_token()?.text_trimmed_range(),
+                    markup! {
+                        "This "<Emphasis>"type"</Emphasis>" qualifier makes all inline "<Emphasis>"type"</Emphasis>" qualifiers useless."
```

Comment:
Out of curiosity, where did you find this terminology? Specifically, the "qualifier" part. I searched in the typescript website, and I couldn't find anything.. 

To me, it seems like very technical wording and not beginner-friendly. I won't stop the PR from merging because, but I think we should:
- find a more beginner-friendly wording
- or explain what "qualifier" means

Reply:
It looks like TypeScript itself may call them "modifiers": https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names

Reply:
> Do you think that `marker` or `keyword` could be better?

`keyword` is definitely beginner friendly. Even `modifier` is a technical word. I would go for `keyword`